peter_lin/midterm/seeds.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'lib/animal'
+require_relative 'lib/client'
+require_relative 'lib/catlady'
+require_relative 'lib/shelter'
+
+def create_shelter_data()
```

Comment:
fyi () after methods are optional in ruby. It's pretty standard to use them if you have arguments to pass into the method, but unusual if there aren't any. If you want to learn some of those best practices, this is pretty thorough: https://github.com/bbatsov/ruby-style-guide